docs/man/keylime_verifier.8.rst

```diff
@@ -0,0 +1,119 @@
+===================
+keylime_verifier.8
+===================
+
+-------------------------
+Keylime verifier service
+-------------------------
+
+:Manual section: 8
+:Author: Keylime Developers
+:Date: September 2025
+
+NAME
+====
+
+keylime_verifier - Keylime verifier service for agent attestation
+
+SYNOPSIS
+========
+
+**keylime_verifier**
+
+(Most operations require root privileges, use with sudo)
+
+DESCRIPTION
+===========
+
+The verifier is a long-running HTTP service that attests registered agents. It uses a database,
+communicates with the registrar, and optionally performs measured boot evaluation and durable
```

Comment:
The verifier does not communicate with the registrar, but accesses the registrar database to obtain agent data